### PR TITLE
Ora2 Changes to work with Ficus Devstack.

### PR DESCRIPTION
**Pull request by staubina**
### Description

Improvements to get ORA2 to be deployable on the Ficus build with Xenial

### Sandbox
- [ ] Build a sandbox for your branch and add a link here

### Testing
- [ ] Unit, integration, acceptance tests as appropriate

### Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [x] Code review: @efischer19 
- [x] Code review: @cahrens 

FYI: @andy-armstrong, @nedbat

### Post-review
- [x] Rebase and squash commits

### CODE REVIEW

**Comment by cahrens** [comment]
👍 Assuming that Eric is able to use the branch and verifies it also works for him. Thanks, AJ!

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "karma-jasmine": "^0.3.6",
     "karma-jasmine-html-reporter": "~0.1",
     "karma-jasmine-jquery": "^0.1.1",
-    "karma-phantomjs-launcher": "^0.1.4",
+    "karma-phantomjs-launcher": "^1.0.4",
```
Comment (efischer19) [inline package.json]: I was able to run `make test-js` successfully after this update

File: scripts/install-wheels.sh [hunk 2]
```diff
@@ -9,10 +9,8 @@ cd `dirname $BASH_SOURCE` && cd ..
 pip install --upgrade pip
 pip install wheel
 
-WHEELHOUSE="scripts/data/wheelhouse"
-
 # Ensure that numpy is installed first; otherwise scipy won't be able to install
-pip install --use-wheel --no-index --upgrade --find-links=$WHEELHOUSE numpy
+pip install --use-wheel numpy==1.6.2
```
Comment (nedbat) [inline scripts/install-wheels.sh]: A minor point: `--use-wheel` is unusual, and not needed.

Comment (efischer19) [inline scripts/install-wheels.sh]: Per https://github.com/edx/edx-ora2/pull/991#issuecomment-288802735, `--only-binary` is faster for Travis builds, but isn't possible without updating dependencies and breaking AI grading. I've filed https://openedx.atlassian.net/browse/TNL-6766 to track removal of that feature.

**Pull request merged**